charts/internal/seed-controlplane/charts/alicloud-cloud-controller-manager/templates/deployment.yaml

```diff
@@ -33,13 +33,18 @@ spec:
 {{ toYaml .Values.podLabels | indent 8 }}
 {{- end }}
     spec:
+      automountServiceAccountToken: false
```

Comment:
Does it need `security.gardener.cloud/trigger` annotaion?

Reply:
No, this setting here is only for preventing mounting the default service account into the CCM pod. It uses a kubeconfig (`--kubeconfig`) to talk to the shoot cluster, hence, it does not need a service account in the seed.

Reply:
Hope I didn't misled you. Maybe I should not ask you question under this statement. I meant for other controllers like CSI and MCM, they have `security.gardener.cloud/trigger` annotation. How about CCM?

Reply:
This is not needed because CCM does not use a `ServiceAccount` of the seed cluster.

Reply:
I understood. Thanks for explanation.